utils/channel_access.py

```diff
@@ -140,6 +160,21 @@ def _values_match(self, pv, expected_value):
                    Expected value: {expected}
                    Actual value: {actual}""".format(PV=pv, expected=expected_value, actual=pv_value)
 
+    def _value_is_close(self, pv, expected_value, delta):
+        """
+        Check pv is close to a value.
+
+        :param pv: name of the pv (no prefix)
+        :param expected_value: value that is expected
+        :param delta: maximum acceptable absolute difference between the values
+        :return: None if they match; error string stating the difference if they do not
+        """
+        pv_value = self.get_pv_value(pv)
+        if abs(pv_value - expected_value) < delta:
+            return None
+        else:
+            return "Expected {expected}: actual {actual}".format(expected=expected_value, actual=pv_value)
```

Comment:
As above. You don't need the `if` here:

```
if not ... :
    return "Expected...
```

If the condition of the `if` isn't met, the function will get to the end and `return None` anyway.